market_maker_keeper/uniswapv2_market_maker_keeper.py

```diff
@@ -84,6 +83,9 @@ def __init__(self, args: list, **kwargs):
         parser.add_argument("--smart-gas-price", dest='smart_gas_price', action='store_true',
                             help="Use smart gas pricing strategy, based on the ethgasstation.info feed")
 
+        parser.add_argument("--geometric-gas-price", dest='geometric_gas_price', action='store_true',
```

Comment:
I think the goal here isn't to offer a new strategy, but to update `SmartGasPrice` to replace `IncreasingGasPrice` with `GeometricGasPrice`, and `ethgasstation-client` with `pygasprice_client`.

Reply:
I was aiming to introduce these changes in parallel so that existing market-maker-keeper implementations could continue using the old client, while new ones switched over - with the testing done with UniswapV2. Removing `ethgasstation-client` would likely need to involve some level of retesting of existing implementations given how important gas estimation is. If that seems worthwhile to switch those over simultaneously too, i'm open to that.